aten/src/ATen/native/Itertools.cpp

```diff
@@ -0,0 +1,55 @@
+#include "ATen/ATen.h"
+#include "ATen/Dispatch.h"
+
+#include <vector>
+
+namespace {
+
+using namespace at;
+
+Tensor _triu_mask(int64_t n, int64_t dims, bool diagonal, const TensorOptions &options) {
+  // get a mask that has value 1 whose indices satisfies i < j < k < ...
+  // or i <= j <= k <= ... (depending on diagonal)
+  TensorOptions opt = std::move(options);
```

Comment:
@ezyang can you double check that this `move` is ok?

Reply:
The move is not necessary (though it does no harm)